pages/about/faq.md

```diff
@@ -1,26 +1,30 @@
 ---
 title: FAQ
 layout: info
+description: Get answers to frequently asked questions about Lawrence Livermore National Laboratory's open source software.
 permalink: /about/faq/
 ---
 
-## {{ page.title }}
```

Comment:
I'll need to fire this up locally and check it out, I seem to recall there being a reason for starting at H2 here in these files, but I can't quite remember.